pom.xml

```diff
@@ -167,10 +170,45 @@
             <artifactId>maven-project-info-reports-plugin</artifactId>
             <version>3.0.0</version>
           </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <!-- A profile for preparing and releasing phoebus.
+     Please refer to http://maven.apache.org/maven-release/maven-release-plugin/index.html 
+      for the detailed description of the steps involved. -->
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-release-plugin</artifactId>
+            <version>2.5.3</version>
+            <configuration>
+              <preparationGoals>exec:exec</preparationGoals>
+              <checkModificationExcludes>
+                <checkModificationExclude>**/.project</checkModificationExclude>
+                <checkModificationExclude>**/.classpath</checkModificationExclude>
+                <checkModificationExclude>pom.xml</checkModificationExclude>
+                <checkModificationExclude>**/phoebus-target/pom.xml</checkModificationExclude>
```

Comment:
Out of curiosity, why do you need to exclude these files from the modification check?